src/Components/Authorization/Mobile/RegisterForm.tsx

```diff
@@ -10,7 +10,7 @@ import {
   inputValidators,
   StyledFacebookButton,
   StyledInput as Input,
-  TOSCheckbox,
+  TOSCheckbox
```

Comment:
@erikdstock - was there no solution to the trailing comma issue? We don't want to remove this setting but rather fix prettier / eslint / tslint so that conflicts don't arise.

Reply:
I'm unclear- do we WANT trailing commas? the specific issue came from object destructuring with `...rest` props at the end which was handled inconsistently from the other pieces.

Reply:
We do, they're super useful when refactoring and this is the setting we have across all other projects.

Reply:
oh, my mistake then. this seems to be the issue: https://github.com/prettier/prettier/issues/3528

Reply:
Yeah, this was fixed in [1.9](https://prettier.io/blog/2017/12/05/1.9.0.html#don-t-print-a-trailing-comma-after-object-rest-spread-3313-https-githubcom-prettier-prettier-pull-3313-by-duailibe-https-githubcom-duailibe)

I'm able to avoid it for now with a `// disable` comment.